### ISSUE TITLE
Proof of work and proof of stake inaccurately described as consensus protocols

### ISSUE THREAD

**Issue by PatrickAlphaC** [ethereum/ethereum-org-website#3198]
> Ethereum, like Bitcoin, currently uses a consensus protocol called Proof-of-work (PoW). This allows the nodes of the Ethereum network to agree on the state of all information recorded on the Ethereum blockchain, and prevents certain kinds of economic attacks.

This is not technically correct. PoW is just a method of choosing a block author to give rewards to, the consensus happens with you combine proof of work with a chain selection rule / block finality, and Bitcoin and Ethereum use the longest chain rule. The combination of PoW and longest chain make up what's called Nakamoto consensus - which is an actual consensus mechanism.

Similarly, ETH 2's proof of stake is likely going to use some other technologies to create a coherent consensus protocol. Proof of stake itself isn't a consensus protocol, it's a catch-all describing a method of choosing block authors - those that have staked.

**Comment by PatrickAlphaC** [comment]
In addition, it would be great to describe the finality of nakamoto consensus as _probabilistic finality_ as it will never be 100% final. There could be a re-org with a chain with 10 confirmations that is suddenly surpassed by a different one with 11 confirmations.

**Comment by minimalsm** [comment]
> This is not technically correct. PoW is just a method of choosing a block author to give rewards to, the consensus happens with you combine proof of work with a chain selection rule / block finality, and Bitcoin and Ethereum use the longest chain rule. The combination of PoW and longest chain make up what's called Nakamoto consensus - which is an actual consensus mechanism.

I'm a bit torn here. Personally I'm weakly in favour of the current wording from a User Experience standing. The heuristic is widely used and it is understood by those in the know that some complexity is hidden to make an already difficult to navigate topic slightly less confusing. At the same time, I think we should strive for absolute truth.

Perhaps an alternative would be to revisit the topic more deeply further into the article?

**Comment by ryancreatescopy** [comment]
I agree with @minimalsm we need to be careful to slowly introduce the technical complexity IMO. We can always use links to help advanced users get to it quicker. @PatrickAlphaC how would you suggest our consensus-mechanisms section gets re-worked to handle your idea?

**Comment by PatrickAlphaC** [comment]
hmm.... perhaps we could just include a section talking about the grander picture? Or we have a page dedicated to the nitty gritty of consensus itself that we link to. I also understand that proof-of-work is sort of an understood term for consensus, while not being totally accurate, and that we want to be as accurate as possible. 

TLDR
If we want to keep UX and accuracy, it might make sense to just have a section describing the nuances and all the intricacies.

**Comment by wackerow** [comment]
@PatrickAlphaC I think you're right that there are nuances to all of the topics on that page that could use some polishing.  I agree that it's a balance between UX and falling down the rabbit hole... I think we should be aware of where the info is presented, starting more user friendly at top and maybe getting deeper into technical nuances deeper down. In this case, I feel referring to "proof-of-work" broadly as the consensus mechanism makes sense, even if churning hashes is only one part of it, but then clarifying in more detail deeper in the page. 

Would you be interested in writing a more technically oriented paragraph or two on this topic? We could potentially place an internal link to the section higher in the page to reference more details information.

**Comment by PatrickAlphaC** [comment]
Always happy to do my open-source duty. 

What do you think, just a paragraph lower that explains:

1. Proof of Work is a Sybil resistance mechanism used to choose a block author
2. Proof of Stake is a different Sybil resistance mechanism used to choose a block author, by users "Staking" some amount to prove they are an honest actor

These combined with something like the "longest chain rule" make for a total consensus mechanism. 

Potentially mention Casper FFG (ETH 2.0 Consensus mechanism). I'm not a casper expert yet... I have yet to read the paper: https://arxiv.org/pdf/1710.09437.pdf

**Comment by wackerow** [comment]
@PatrickAlphaC Yeah, that sounds great!... were you thinking one paragraph for each page (PoW and PoS)? Any thoughts on where in the PoW page would work best for this?

I certainly would certainly love to see expansion of the PoS docs, especially given the changed the merge will bring. That paper is solid, but obviously super technical and can take a bit to process it all. Made me realize that we don't appear to have a link to it on the page, maybe we could at least link to it from the PoS page in the meantime.

**Comment by PatrickAlphaC** [comment]
PR https://github.com/ethereum/ethereum-org-website/pull/3353
